src/openforms/js/components/admin/form_design/registrations/zgw/ZGWOptionsForm.js

```diff
@@ -41,6 +41,7 @@ const ZGWOptionsForm = ({name, label, schema, formData, onChange}) => {
         zaakVertrouwelijkheidaanduiding: '',
         medewerkerRoltype: '',
         propertyMappings: [],
+        product: '',
```

Comment:
if we only support a single product selection, why do we have to pass a list of product URLs to the case creation? wouldn't it make more sense in the backend to only accept a single URL `str` argument instead of a list then?

Reply:
The backend already expected a single product, but i've changed it so that the `ZakenClient` turns it into an array before assigning it to the `productenOfDiensten` attribute